ramls/exchange-rate.raml

```diff
@@ -0,0 +1,64 @@
+#%RAML 1.0
+
+title: Exchange rate
+version: v1
+protocols: [ HTTP, HTTPS ]
+baseUri: https://github.com/folio-org/mod-finance
+
+documentation:
+  - title: Exchange rate API
+    content: This documents the API calls that can be made to get exchange rates
+
+types:
+  exchangeRate: !include acq-models/mod-finance/schemas/exchange_rate.json
+  error: !include raml-util/schemas/errors.schema
+  currency_code:
+      type: string
+      pattern: ^[A-Z]{3}$
+
+traits:
+  searchable: !include raml-util/traits/searchable.raml
+  language: !include raml-util/traits/language.raml
+
+/finance/exchange-rate:
+  displayName: Exchange rate
+  description: Exchange rate API
+  is: [language]
```

Comment:
As far as you use regex for currency_code it makes sense to use validate trait here. So you will have 422 if illegal request, 400 if one of the currency codes passed is not valid and 404 if no such rate is available. This is connected with https://github.com/folio-org/mod-finance/pull/53/files#r406768993. cc @folio-org/acquisitions